rest-server/auth-server/src/main/resources/application.conf

```diff
@@ -9,10 +9,39 @@ auth-server {
 
   mongo-db {
     uri = "mongodb://localhost:27017"
-    db-name = "harness_auth"
+    db-name = "harness_auth_test"
   }
 
+  clients = [{
+    id = "harness"
+    password = "mwjmo4fwC9qFW081gm5vwsmAC4Z2747XEU8eeV6dzUzjW"
+  }]
+
+  role-sets = [
+    {
+      id = "admin"
+      roles = [
+        "user_create",
+        "user_permissions",
+        "engine_create"
+      ]
+    },
+    {
+      id = "client"
+      roles = [
+        "engine_read",
+        "event_create",
+        "event_read",
+        "event_modify",
+        "query_create",
+        "query_modify",
+        "query_read"
+      ]
+    }
+  ]
+
   access-token-ttl = 3600000 // 1 hour
```

Comment:
Is an hour necessary? Could this be more like a day? Input can come in very fast, in bursts, and every time a new access token is required it will slow down the request. The access token is made to speed up auth so the credentials do not need to be checked on every request, right?

Reply:
strictly speaking the only REST defined is:
```    
"engine_read",
"event_create",
"query_create",
```

Since events and queries are not persisted they cannot be read or modified.

Reply:
> Is an hour necessary? Could this be more like a day?
It's not my magic number, it's recommendations of oauth2 spec https://tools.ietf.org/html/rfc6750#section-5.3

Reply:
@pferrel updated rolesets